demos/demo_solo12_from_yaml.cpp

```diff
@@ -45,30 +46,35 @@ int main()
             {
                 if (!is_calibrated)
                 {
-                    is_calibrated = calib_ctrl->Run();
+                    is_calibrated = calib_ctrl->RunAndGoTo(pos_init);
                     if (is_calibrated)
                     {
                         std::cout << "Calibration is done." << std::endl;
                     }
                 }
                 else
                 {
-                    // Run the main controller.
-                    auto pos = robot->joints->GetPositions();
-                    auto vel = robot->joints->GetVelocities();
-                    // Reverse the positions;
-                    for (int i = 0; i < 12; i++)
-                    {
-                        torques[i] = -kp * pos[i] - kd * vel[i];
-                    }
-                    robot->joints->SetTorques(torques);
+                    // Set all command quantities
+                    robot->joints->SetTorques(Eigen::VectorXd::Zero(12));
+                    robot->joints->SetDesiredPositions(pos_init);
+                    robot->joints->SetDesiredVelocities(
+                        Eigen::VectorXd::Zero(12));
+                    robot->joints->SetPositionGains(kp *
+                                                    Eigen::VectorXd::Ones(12));
+                    robot->joints->SetVelocityGains(kd *
+                                                    Eigen::VectorXd::Ones(12));
                 }
             }
 
             // Checks if the robot is in error state (that is, if any component
             // returns an error). If there is an error, the commands to send
             // are changed to send the safety control.
-            robot->SendCommand();
+            if (!robot->SendCommand())
+            {
+                throw std::runtime_error(
+                    "Error when sending the command. Please check error "
+                    "messages.");
+            }
```

Comment:
Same comment about not-throwing an exception here as in previous comment.

Reply:
I agree, I will remove it like for the other demo.